ktor-utils/js/src/io/ktor/util/PlatformUtilsJs.kt

```diff
@@ -5,17 +5,16 @@
 
 package io.ktor.util
 
-public actual object PlatformUtils {
-    public actual val IS_BROWSER: Boolean = js(
-        "typeof window !== 'undefined' && typeof window.document !== 'undefined' || typeof self !== 'undefined' && typeof self.location !== 'undefined'" // ktlint-disable max-line-length
-    ) as Boolean
+internal actual val PlatformUtils.isDevelopmentMode: Boolean
+    get() = false
 
-    public actual val IS_NODE: Boolean = js(
-        "typeof process !== 'undefined' && process.versions != null && process.versions.node != null"
-    ) as Boolean
+internal actual val PlatformUtils.isNewMemoryModel: Boolean
+    get() = true
 
-    public actual val IS_JVM: Boolean = false
-    public actual val IS_NATIVE: Boolean = false
-    public actual val IS_DEVELOPMENT_MODE: Boolean = false
-    public actual val IS_NEW_MM_ENABLED: Boolean = true
-}
+public actual val PlatformUtils.platform: Platform
+    get() {
+        val hasNodeApi = js(
+            "typeof process !== 'undefined' && process.versions != null && process.versions.node != null"
```

Comment:
After testing this in Electron the check turned out to return false even though we have the Node API. In Electron at this point there can be two `process` objects and the correct one is defined on `window.process`. So we'd have to first detect which process object to use and after that the remaining `.versions.node` check can be done.

Reply:
Do you mind creating a ticket so that it doesn't get lost?

Reply:
Yes sure: https://youtrack.jetbrains.com/issue/KTOR-5906/Electron-Node.js-detection-doesnt-work-correctly